src/main/scripts/releases/2.11/legacy/add20MillionTreesProgramToDatabase.js

```diff
@@ -0,0 +1,78 @@
+load('../../../utils/uuid.js');
```

Comment:
Although there is a program configured in the programs-model with this name, it is not used by any projects in MERIT.  Instead the sub-program of the National Landcare program with the same name is used.
Delete this as there is no need to create unused programs in the database.